src/common/Serializable.h

```diff
@@ -77,4 +77,59 @@ class Serializable {
   }
 };
 
+// This is a temporary class for use with data blocks
+class Serializable2 {
```

Comment:
Can change the Serializable2 to a more meaningful name?

Reply:
Changed to `SerializableDataBlock`